app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,78 @@
+require 'pry'
+
+class TweetsController < ApplicationController
+  get '/tweets' do
+    if !!session[:user_id]
+      @user = User.find(session[:user_id])
+      @tweets = Tweet.all
+      erb :"/tweets/tweets"
+    else
+      redirect '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if !!session[:user_id]
+      @user = User.find(session[:user_id])
+      erb :"/tweets/create_tweet"
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/tweets' do
+    if !params[:content].empty?
+      @user = User.find(session[:user_id])
+      @tweet = Tweet.create(content: params[:content])
+      @user.tweets << @tweet
+      redirect '/tweets'
+    else
+      redirect '/tweets/new'
+    end
+  end
+
+  get '/tweets/:id' do
+    if !!session[:user_id]
+      @tweet = Tweet.find(params[:id])
+      erb :"/tweets/show_tweet"
+    else
+      redirect '/login'
+    end
+  end
+
+  get '/tweets/:id/edit' do
+    if !!session[:user_id]
+      @user = User.find(session[:user_id])
+      @tweet = Tweet.find(params[:id])
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?